src/client/editor/inspector.js

```diff
@@ -194,22 +204,30 @@ let inspector = function() {
 
 			// Width
 			shapeChangeFunctions.width = function() {
-				inspector.selected.setWidth(parseFloat(inspector.elements.shapeProperties.input.width.value));
+				let value = inspector.elements.shapeProperties.input.width.valueAsNumber;
+				if(isNaN(value)) value = 1;
```

Comment:
You can't do that same trick here though (because if `value === 0`, it will be still be a falsey value)
Additionally, should we verify these values to make sure they're not 0 or negative?